backend/api/routes.py

```
@@ -0,0 +1,315 @@
+            job_id=job.id,
+            status="pending",
+            estimated_duration=job.estimated_duration,
+            message="Job created successfully. Processing will begin shortly.",
+        )
+
+    except DemucsException as e:
+        logger.error("stemify_job_creation_failed", error=e.message, exc_info=True)
+        raise HTTPException(
+            status_code=e.status_code,
+            detail=e.to_dict(),
+        )
```

Ruff (0.13.1)

Within an `except` clause, raise exceptions with `raise ... from err` or `raise ... from None` to distinguish them from errors in exception handling

(B904)